### PR TITLE
test: put the new DHE auto test in the correct place

### PR DESCRIPTION
**Pull request by paulidale**
This fixes the daily build failure.

- [ ] documentation is added or updated
- [x] tests are added or updated

### CODE REVIEW

File: test/recipes/80-test_ssl_old.t [hunk 1]
```diff
@@ -527,11 +527,14 @@ sub testssl {
                 if $no_tls1 || $provider eq "fips";
 
         SKIP: {
-            skip "skipping anonymous DH tests", 1
+            skip "skipping anonymous DH tests", 2
                 if ($no_dh);
 
             ok(run(test([@ssltest, "-v", "-bio_pair", "-tls1", "-cipher", "ADH", "-dhe1024dsa", "-num", "10", "-f", "-time"])),
                'test tlsv1 with 1024bit anonymous DH, multiple handshakes');
+
```
Comment (slontis) [inline test/recipes/80-test_ssl_old.t:535]: Does this need its own skip section since it is both a if (no_dh) AND (no_psk)

Comment (paulidale) [reply test/recipes/80-test_ssl_old.t:535]: Yes it does as I later discovered...
The 1.1.1 PR is breaking for other reasons...

File: test/recipes/80-test_ssl_old.t [hunk 2]
```diff
@@ -563,6 +560,14 @@ sub testssl {
 	  }
 	}
 
+        SKIP: {
+            skip "skipping auto DH tests", 1
```
Comment (slontis) [inline test/recipes/80-test_ssl_old.t:564]: auto DH psk tests?

**Review by slontis — APPROVED** [review]
Provided it passes the tests

File: test/recipes/80-test_ssl_old.t (hunk 2 above)
Comment (paulidale) [reply test/recipes/80-test_ssl_old.t]: Yeah, better

**Comment by paulidale** [comment]
Merged, thanks for the reviewing.

**Pull request closed without merging**